common/djangoapps/student/models.py

```diff
@@ -1506,10 +1506,13 @@ def change_mode(self, mode):
         """Changes this `CourseEnrollment` record's mode to `mode`.  Saves immediately."""
         self.update_enrollment(mode=mode)
 
-    def refundable(self):
+    def refundable(self, user_already_has_certs_for=None):
         """
         For paid/verified certificates, students may receive a refund if they have
         a verified certificate and the deadline for refunds has not yet passed.
```

Comment:
We talked before about disallowing refunds if you've been issued a verified certificate. I don't think the comment is accurate, but then again, my memory of the requirements doesn't necessarily have to align with what she shipped.